dpc-sdp/ripple-framework
# Update to Nuxt RC 13 and add publish steps

**Issue**:

### What I did

- Add steps to publish to NPM using lerna 
- Add circleci workflows for alpha tag release on merge to develop and latest version release on merge to master
- Update to Nuxt 3.0.0 RC 13
- Update Vue to 3.2.41 to track Nuxt dependency version
- Move nuxt-app config to ripple-ui-core for postcss 
- Move ripple-social-sharing plugin to ripple-ui-core
- Release alpha versions of packages for testing reference site

### How to test

- Merge PR to develop
- Check CI passes and publishes to NPM - https://www.npmjs.com/package/@dpc-sdp/ripple-tide-api
- Merge develop to master
- Check CI passes and publishes to NPM with prod version update

### Checklist




#### For all PR's


#### For new components only
